remme/tp/account.py

```diff
@@ -97,45 +96,73 @@ def _transfer(self, context, public_key, transfer_payload):
         References:
             - https://github.com/Remmeauth/remme-core/blob/dev/remme/genesis/__main__.py
         """
-        address = self.make_address_from_data(public_key)
+        if transfer_payload.sender_account_type == TransferPayload.ACCOUNT:
+            address = self.make_address_from_data(public_key)
+
+        else:
+            address = NodeAccountHandler().make_address_from_data(public_key)
 
         return self._transfer_from_address(context, address, transfer_payload)
 
+    def is_address_account_type(self, address):
+        """
+        Check if address is account address type.
+        """
+        if address.startswith(self._prefix):
```

Comment:
Why not just `return a or b or c`?

Reply:
@eugene-babichenko, didn't get you, can you rephase?

Reply:
Particularly

```python
def is_address_account_type(self, address):
    return address.startswith(self._prefix) or \
        address.startswith(NodeAccountHandler()._prefix) or \
        address == ZERO_ADDRESS
```

Reply:
@eugene-babichenko, looks a bit ugly that mine approach, imho. But done.